diff -urN libfreenect2-0.2.0/src/frame_listener_impl.cpp libfreenect2-0.2.0-new/src/frame_listener_impl.cpp
--- libfreenect2-0.2.0/src/frame_listener_impl.cpp	2016-04-27 22:34:51.000000000 +0200
+++ libfreenect2-0.2.0/src/frame_listener_impl.cpp	2018-09-08 17:24:57.801977902 +0200
@@ -109,7 +109,7 @@
 #ifdef LIBFREENECT2_THREADING_STDLIB
   libfreenect2::unique_lock l(impl_->mutex_);
 
-  auto predicate = std::bind(&SyncMultiFrameListenerImpl::hasNewFrame, impl_);
+  auto predicate = [this]{ return impl_->hasNewFrame(); };
 
   if(impl_->condition_.wait_for(l, std::chrono::milliseconds(milliseconds), predicate))
   {
diff -urN libfreenect2-0.2.0/src/logging.cpp libfreenect2-0.2.0-new/src/logging.cpp
--- libfreenect2-0.2.0/src/logging.cpp	2016-04-27 22:34:51.000000000 +0200
+++ libfreenect2-0.2.0/src/logging.cpp	2018-09-08 17:26:17.760977597 +0200
@@ -35,7 +35,6 @@
 #ifdef LIBFREENECT2_WITH_PROFILING
 #include <vector>
 #include <numeric>
-#include <functional>
 #include <cmath>
 #endif
 
@@ -276,7 +275,7 @@
     size_t n = v.size();
     double mean = sum / n;
     std::vector<double> diff(n);
-    std::transform(v.begin(), v.end(), diff.begin(), std::bind2nd(std::minus<double>(), mean));
+    for (size_t i = 0; i < n; ++i) diff[i] = v[i] - mean;
     double sqsum = std::inner_product(diff.begin(), diff.end(), diff.begin(), 0.0);
     double std = std::sqrt(sqsum / (n-1));
 
